.travis.yml

```diff
@@ -8,6 +8,7 @@ matrix:
    - r: 3.3
    - r: 3.4
    - r: 3.5
+     env: VCR_TURN_OFF=true
```

Comment:
It's useful I think to have one build with vcr turned off. That said it might be better to have it run regularly, maybe easier if this repo CI switches to GH Actions (where a workflow could be run every day or so).